src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -184,4 +195,34 @@ public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
         return Objects.hash(persons, tags);
     }
+
+    /**
+     * Removes the particular tag for all people in the AddressBook }.
+     */
+    public void removeTag(Tag tag) throws DuplicatePersonException, PersonNotFoundException {
```

Comment:
There is no associated update to user guide for this feature. Remember an enhancement or change in the feature should ALWAYS be accompanied by an update to the test and documentation